diff --git a/src/snappy/CMakeLists.txt b/src/snappy/CMakeLists.txt
--- a/src/snappy/CMakeLists.txt
+++ b/src/snappy/CMakeLists.txt
@@ -26,7 +26,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 project(Snappy VERSION 1.1.9 LANGUAGES C CXX)
 
 # C++ standard can be overridden when this is used as a sub-project.
