diff --git a/src/libpng/CMakeLists.txt b/src/libpng/CMakeLists.txt
--- a/src/libpng/CMakeLists.txt
+++ b/src/libpng/CMakeLists.txt
@@ -29,8 +29,8 @@
 # This code is released under the libpng license.
 # For conditions of distribution and use, see the disclaimer
 # and license in png.h
 
-cmake_minimum_required(VERSION 3.1)
-cmake_policy(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
+cmake_policy(VERSION 3.5)
 
 project(libpng C ASM)
 enable_testing()
